.circleci/config.yml

```diff
@@ -26,6 +26,7 @@ jobs:
             pip install . && touch .venv/lib/python3.7/site-packages/snowflake/__init__.py
             pip install rpy2 pandas
             pip install mypy
+            pip install slackclient
```

Comment:
is there a need for this? circleci should not need this, nor the other installs above it?